test/test_processor/test_config.py

```diff
@@ -428,6 +428,11 @@ def test_fill_in_fixed_from_settings(self):
                                                         fixed)
         assert fixed == [{'n_sersic': 4.}]
 
+        fixed2 = [{}, {}, {}, {}]
```

Comment:
This line is redundant.

Reply:
Do you mean just line 431 or the entire fixed2 section (line 431 to 434)? I am quite sure line 431 is necessary if we decide to  add line 432 to 434.

Reply:
O I see that `self.config3.fill_in_fixed_from_settings()` requires `fixed2` as  an argument, which I missed before. This is fine.